### DISCUSSION TITLE
This Battle Pass is not worth it!

### DISCUSSION THREAD

**Discussion by Tausert** [ValveSoftware/Dota2-Gameplay#3509] [General]
This Battle Pass is not worth it and is a disappointment to users.
The Battle Pass 2022 is the sign of the decline of the Dota 2 content developers, designers and marketing team.
I play Dota since 2006 in Dota 1 and Dota 2 since early 2012 when I got the beta key. I've bought a lot of their content and have been a loyal follower and consumer of their Battle Pass, since the first Compendium came out in 2013. And I have never felt so disappointed and ripped off for buying a Battle Pass as I feel today.

Your Battle Pass 2022 has only brought us:

- Incomplete content.
- Fewer ways to level up for free.
- Only 2 immortal chests instead of 3.
- The removal of gold coins.
- The elimination of the Crystal roulette.
- The elimination of the side store.
- Only 2 arcana instead of 3.
- Two chests with poorly designed sets.
- A reward straight full of unnecessary and worthless items such as balloons, voice wheels, sprays, emoticons, bubbles, gloves and cavern tools. Which is a clear attempt to trick our eyes by stuffing the BP with pure junk.
- The ineffectiveness and non-existent value of the clans.

In short, you have just given us a Battle Pass full of useless and worthless content for us, with much fewer immortals and ways to earn good worthwhile rewards, at an even higher price due to the dollar exchange rate.

Many of us are veteran players and collectors who like to invest money in the game and get items that are really worthwhile. We want to feel pampered as your customers, we want to receive good content and feel that we matter to you. But you don't have the slightest idea of what valuable content is and much less of how to know how to retain your old customers and how to catch new potential customers.

For many of us, this will be the last Battle Pass we will buy, because it's honestly not worth spending money on it anymore.

**Comment by onesmartsheep1** [comment]
I really appreciate the points you have discussed in the piece of writing. Commendable[.](https://onesmartsheep.com/) Indeed, it isn't worth spending money on.

**Comment by gaia1991** [comment]
me too very disappointing on these game now. planning to quit and uninstall game

**Comment by copypastelist** [comment]
I could not find myself more disappointing and dejecting[.](https://copypastelist.co/) It is really not worthwhile.
